.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,44 @@
+name: "\U0001F41E Bug report"
+description: Report an issue with Sheepdog
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue, tell us in the description. Thanks!
+      placeholder: Bug description
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a link to a repo or REPL that can reproduce the problem you ran into. If a report is provided without an easy way to reproduce the bug, it's likely we will not be able to help.
+      placeholder: Reproduction
+    validations:
+      required: true
+  - type: dropdown
+    id: severity
+    attributes:
+      label: Severity
+      description: Select the severity of this issue
+      options:
+        - annoyance
+        - blocking all usage of Sheepdog
```

Comment:
i would add some other option here...we already see this a lot in svelte that people post a very minimal bug with a skewed severity. We need something that is serious but doesn't block everything

Reply:
What other options do you think would fit here? The other option is to simply remove the input

Reply:
I think something like

- annoyance 
- serious but I have a workaround
- blocking me on a specific feature
- blocking all usage